internal/distro/fedora/distro.go

```diff
@@ -591,6 +592,10 @@ func (t *imageType) PayloadPackageSets() []string {
 	return []string{blueprintPkgsKey}
 }
 
+func (t *imageType) PackageSetsChains() map[string][]string {
+	return t.packageSetChains
+}
+
```

Comment:
Something must have gotten mixed up during the rebase because the message for commit 0408f2322e8cbded128772990cdff372215fdfe8 doesn't match the changes.

Reply:
Indeed. This is odd. I'll take a look.

Reply:
Looks like the commit I added to update the commid ID for fedora in the `Schutzfile` was impacted when I rebased and included this later on commit:
https://github.com/osbuild/osbuild-composer/commit/e3f812f6fd60386502f921fa829debf869110ec5
I can change the description of the commit since it does not apply anymore.

Reply:
I think that the part adding the `PackageSetsChains()` method should be actually squashed to the commit that replaced the v1 Fedora distro implementation with v2. Because without this method, the `imageType` struct won't actually implement all methods of the `ImageType` interface and the code would be basically broken in that version.

Reply:
I'll squash the commits for the changes in the code related to `PackageSetChains()`.